wellcomecollection/wellcomecollection.org
# Rely on Prismic filters to fetch pages based on tagged siteSection

## What does this change?

Had a chat with Robert regarding [the comment](https://github.com/wellcomecollection/wellcomecollection.org/pull/11248#issuecomment-2418935762) he left in #11248 and it's a really good idea/probably the main way to implement this change.

Relates to https://github.com/wellcomecollection/wellcomecollection.org/issues/11206 // For https://github.com/wellcomecollection/wellcomecollection.org/issues/11072

So now we're filtering directly at the fetch level, which removes the need to fetch the document to then make sure it's rendering at the correct route. 
I've added the possibility to pass in "orphan" as site section, that safe guards pages rendering at the `/[uid]` / orphan route (otherwise they don't get filtered and they're all allowed to render there). 

Once we're happy with this I can write tests that will ensure this works as expected moving forward.

## How to test

[See How to test section in parent PR](https://github.com/wellcomecollection/wellcomecollection.org/pull/11248#issue-2575629819), it should get the exact same results.

## How can we measure success?

More efficient queries!

## Have we considered potential risks?
N/A
